analyses/independent-samples/run-independent-samples.sh

```diff
@@ -14,6 +15,11 @@ RUN_FOR_SUBTYPING=${OPENPBTA_BASE_SUBTYPING:-0}
 # Set the working directory to the directory of this file
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
+if [[ "$RUN_FOR_SUBTYPING" -eq "1" ]]
+then
+# Run rna-seq only
+Rscript 04-generate-independent-specimens-rnaseq-release.R
```

Comment:
```suggestion
  # Run rna-seq only
  Rscript 04-generate-independent-specimens-rnaseq-release.R
```

Indent for easier to read.

Could you also indent after `else` and before `fi`?

Reply:
done